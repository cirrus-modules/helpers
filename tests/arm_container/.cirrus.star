load("../../lib.star", "arm_container", "task")

def main(ctx):
    return [
        task("task with a default ARM container", arm_container()),
        task("task with a container instantiated from debian:latest image", arm_container("debian:latest")),
        task("task with a container built from Dockerfile", arm_container(dockerfile = "./ci/Dockerfile")),
        task("task with a high-performance container", arm_container(cpu = 16, memory = 16384)),
    ]
