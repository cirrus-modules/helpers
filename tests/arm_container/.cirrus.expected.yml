task:
  name: task with a default ARM container
  arm_container:
    cpu: 2
    memory: 4096
task:
  name: task with a container instantiated from debian:latest image
  arm_container:
    image: debian:latest
    cpu: 2
    memory: 4096
task:
  name: task with a container built from Dockerfile
  arm_container:
    dockerfile: ./ci/Dockerfile
    cpu: 2
    memory: 4096
task:
  name: task with a high-performance container
  arm_container:
    cpu: 16
    memory: 16384
